hack/rbacgenerator/function.js

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think so. Better than it only existing in a random Azure Functions deployment. It doesn't have to be permanent. Once ARM team implements `uniqueGUID` all of this can go away.